.github/workflows/common-app-build.yml

```diff
@@ -42,6 +42,10 @@ jobs:
           fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
           set-safe-directory: true
 
+      # make sure the renv directory is created
+      - name: Ensure renv library directory exists
+        run: mkdir -p ./renv/library
```

Comment:
neskusal si to s tymi pravami?

Reply:
skusil som, ale práva bohužiaľ nepomohli.... 

@cvoky @PetoLau 
Za chybu, ktorá sa týkala toho, že nevedelo nájsť nainštalovaný package "DeltaForecastUberModelAPI" mohol tento  update v RENV 0.16.0: `The renv sandbox is once again enabled by default. On Unix, the sandbox is now created by default within the project’s renv/sandbox library. On Windows, the sandbox is created within the R session’s tempdir(). The path to the renv sandbox can be customized via the RENV_PATHS_SANDBOX environment variable if required.
`
Preto dávalo libPaths ako: 
```
[1] "/root/.cache/R/renv/library/app-d4a5a8f5/R-4.3/x86_64-pc-linux-gnu"
[2] "/root/.cache/R/renv/sandbox/R-4.3/x86_64-pc-linux-gnu/25ebdc09"
```

namiesto tohoto: 
```
[1] "/opt/app/renv/library/R-4.3/x86_64-pc-linux-gnu"
[2] "/tmp/RtmppJG2T4/renv-system-library"

```

po pridaní  `ENV RENV_CONFIG_SANDBOX_ENABLED=FALSE` už naštartovanie appky v containeri funguje v pohode. Musím avšak ešte zistiť, ako správne nastaviť cache -> lebo teraz pri každom builde stále nanovo sťahuje všetky packages a build trvá cez 15 min...

Reply:
super, tj uzitocne zistenie